azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -123,6 +123,16 @@ private void sendSlaAlertEmail(final SlaOption slaOption, final String slaMessag
     }
   }
 
+  private String getJobOrFlowName(SlaOption slaOption) {
```

Comment:
Could you add unit tests? 
The email component is not unit test friendly yet. We would like to refactor it.
I think components of this change such as this method can be unit tested at least.

Reply:
Umm.. do you mean that _you_ will refactor it to be unit test friendly (before writing a test)?

Reply:
@jamiesjc plans to refactor the email component.
I am suggesting that even before that takes place, new changes should have automated test coverage as much as possible.
It's on the todo list to get code coverage report integrated with the review process.